postgres/dev.sls

```diff
@@ -45,9 +45,10 @@ postgres_maxfiles_limits_conf:
   file.managed:
     - name: /Library/LaunchDaemons/limit.maxfiles.plist
     - source: salt://{{ tpldir }}/templates/limit.maxfiles.plist
+    - template: jinja
     - context:
-      soft_limit: {{ postgres.limits.soft or postgres.limits.hard }}
-      hard_limit: {{ postgres.limits.hard or postgres.limits.soft }}
+      soft_limit: {{ postgres.limits.soft | default(postgres.limits.hard, true) }}
+      hard_limit: {{ postgres.limits.hard | default(postgres.limits.soft, true) }}
```

Comment:
@noelmcloughlin Since we have added the default values to the OS family map, they would be merged to the `postgres` dictionary anyway if nothing set in the Pillar. So the `default` filter becomes unnecessary.

Reply:
Done, thanks for the review @vutny